### PR TITLE
Add Mixpanel and General Updates

### PR DESCRIPTION
**Pull request by ephan**
- Added Mixpanel board under Analytics tab
 - Updates to consolidate error handling and displaying with toast and react query
 - Displaying & inputting Ada instead of lovelace
 - Moved mixpanel project token to env

Screen Shot 2024-07-10 at 5 24 38 PM

### CODE REVIEW

File: src/components/PlaceBid/PlaceBid.tsx [hunk 1]
```diff
@@ -49,16 +54,19 @@ export const PlaceBidForm = ({
     );
     console.log({ placeBidResponse });
```
Comment (ephan) [inline src/components/PlaceBid/PlaceBid.tsx:55]: Keeping console logs in for debugging & testing API calls

File: src/hooks/api/claim.ts [hunk 2]
```diff
@@ -20,15 +20,20 @@ export const useClaimAuctionLotBidder = (config: ContractConfig) => {
...
+      toast.success('Auction lot claimed successfully.');
+      mixPanel?.track('ClaimLotSucceeded', {
+        userType: 'bidder',
       });
-      return claimAuctionLotBidderResponse;
     },
-    onSuccess: (_, variables) => {
-      mixPanel && mixPanel.track('Claimed Auction Lot Bidder');
+    onError: (error) => {
+      console.error('Error claiming auction lot bidder', error);
```
Comment (SamDelaney) [inline src/hooks/api/claim.ts:35]:
```suggestion
      console.error('Error claiming auction lot as winning bidder.', error);
```

File: src/hooks/api/claim.ts [hunk 3]
```diff
@@ -40,10 +45,24 @@ export const useClaimAuctionLotSeller = (config: ContractConfig) => {
...
+        claimAuctionLotSellerResponse
+      );
+
+      return claimAuctionLotSellerValidated;
     },
-    onSuccess: (_, variables) => {
-      mixPanel && mixPanel.track('Claimed Auction Lot Seller');
+    onSuccess: () => {
+      toast.success('Auction lot claimed successfully.');
+      mixPanel?.track('Claimed Auction Lot Seller');
```
Comment (SamDelaney) [inline src/hooks/api/claim.ts:61]:
```suggestion
      mixPanel?.track('Reclaimed Auction Lot as Seller');
```

File: src/hooks/api/discoverSellerSignature.ts [hunk 4]
```diff
@@ -23,14 +26,19 @@ export const useDiscoverSellerSignature = (
       console.log('useDiscoverSellerSignature');
       console.log({ discoverSellerSignatureParams: params });
 
+      console.log({ sellerSigParams: params });
```
Comment (SamDelaney) [inline src/hooks/api/discoverSellerSignature.ts:29]: logging the same thing twice? Looks like this file could use some cleanup on logs

File: src/hooks/api/moveBidL2.ts [hunk 5]
```diff
@@ -15,21 +16,17 @@ export const useMoveBidL2 = (config: ContractConfig) => {
...
       }
     },
     onSuccess: () => {
       toast.success('Bidding moved to L2');
-      mixPanel && mixPanel.track('Bidding moved to L2');
+      mixPanel?.track('Bidding moved to L2');
     },
     onError: (error) => {
-      toast.error(error.message);
+      toast.error(` Moving bid to L2 failed: ${error.message}`);
```
Comment (SamDelaney) [inline src/hooks/api/moveBidL2.ts:29]: OCD moment

```suggestion
      toast.error(`Moving bid to L2 failed: ${error.message}`);
```

File: src/index.tsx [hunk 6]
```diff
@@ -8,9 +8,9 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  // <React.StrictMode>
```
Comment (SamDelaney) [inline src/index.tsx:11]: Why drop strictmode?

**Review by SamDelaney — CHANGES_REQUESTED** [review]
A lot of this is pretty nitpicky, sorry if that's annoying. I mostly care about making sure that logs aren't overused or completely arcane to new users.

File: src/components/AuctionDetail/AuctionSubDetail.tsx [hunk 7]
```diff
@@ -1,7 +1,11 @@
...
 import { ListBulletIcon } from '@radix-ui/react-icons';
 import { AuctionInfo } from 'hydra-auction-offchain';
-import { numberWithCommas } from 'src/utils/currency';
+import {
+  ADA_CURRENCY_SYMBOL,
+  formatLovelaceToAda,
+  numberWithCommas,
+} from 'src/utils/currency';
 
 const MOCK_NFT_DESCRIPTION = 'Mock NFT Description 🐶';
```
[the remark this answers is not in the document]
Comment (SamDelaney) [reply src/components/AuctionDetail/AuctionSubDetail.tsx:10]: To be clear I love 🐶 no changes required here ❤️

File: src/index.tsx (hunk 6 above)
Comment (ephan) [reply src/index.tsx:11]: Dealing with the too many requests error from Blockfrost

**Review by SamDelaney — APPROVED** [review]
LGTM!

**Pull request merged**